spec/factories/collections.rb

```diff
@@ -12,11 +9,7 @@
     association :owner, factory: :user
 
     project_ids do
-      if build_projects
-        [ create(:project).id ]
-      else
-        (1..10).to_a.sample(2)
-      end
+      [ create(:project).id ]
```

Comment:
Collections will always build a project now (and all the cascading project resources) and if heavily used it can slow the tests down. I'm not so worried but if known maybe pass in the project when creating the factory? Also should we rename this to projects association?

Reply:
I considered that but there's no real way to search for locations where it does need the project to be built. Perhaps in a followup PR?

Reply:
up to you, it was transient before so only those marked to build it required it by default.